src/client.ts

```diff
@@ -194,7 +194,10 @@ export class Client {
       onEnd<OperationResult>(() => this.onOperationEnd(operation))
     );
 
-    return this.suspense && operationName !== 'subscription'
+    if (operation.context.suspense === undefined)
+      operation.context.suspense = this.suspense;
+
+    return operation.context.suspense && operationName !== 'subscription'
```

Comment:
to be honest, we could save some bytes by changing this to `operationName === 'query'`

Reply:
Or:

```
operationName === 'query' &&
(operation.context.suspense === true || this.suspense)